nnpdf_data/nnpdf_data/commondata/ATLAS_Z0J_8TEV/metadata.yaml

```diff
@@ -102,7 +107,7 @@ implemented_observables:
         description: Variable k3
         label: k3
         units: ''
-    file: kinematics_PT-M.yaml
+    file: kinematics_legacy_PT-M.yaml
```

Comment:
Just for reference, we don't need to keep the legacy kinematics, you can just remove these files.
The information should be the same as in the new ones, and the new ones have all the info we need (name of the variables mainly)

Reply:
Thank you. However, I'd say that the information is not the same. The "old" kinematics used the mid value of the bin (squared), while the new one that I'm going to implement will contain min and max value of the bin (non-squared). If you say that it doesn't make any difference, and we can get rid of the old kinematics, I will remove the latter and implement the new kinematics as mentioned above.

Reply:
Yes, you can remove the old one. You can fill min, max and the midpoint will be filled automatically (and should be the same as before).

The reason only the midpoint was used before is that validphys could not deal with that extra information, but having the min-max is much better because then we can use these datasets to autogenerate runcards (for instance the NNLOJET runcards need this information)

Reply:
Ok, thank you so much. This is indeed what I was trying to understand. I go on.